f8a_report/main.py

```diff
@@ -6,9 +6,17 @@
 from v2.report_generator import StackReportBuilder
 from manifest_helper import manifest_interface
 import os
+import requests
 
 logger = logging.getLogger(__file__)
 
+_INVOKE_API_WORKERS = os.environ.get("INVOKE_API_WORKERS", "1")
+_WORKER_ADMINISTRATION_REGION = os.environ.get("WORKER_ADMINISTRATION_REGION", "api")
+_INGESTION_SERVICE_HOST = os.environ.get("INGESTION_SERVICE_HOST", "bayesian-jobs")
+_INGESTION_SERVICE_PORT = os.environ.get("INGESTION_SERVICE_PORT", "34000")
+_INGESTION_SERVICE_ENDPOINT = "api/v1/ingest-epv"
+_APP_SECRET_KEY = os.getenv('APP_SECRET_KEY', 'not-set')
+
```

Comment:
What is Point of Using `_` (Underscore)

Reply:
It was suggested by Arun in few of my PRs to make global variables/function private, if they are not used in any other places. Earlier I used to name global variable without _ (underscore), but he asked me to change it to _<VARIABLE_NAME> and make it private. Please refer to his comment on my other PR.

https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/911#discussion_r463095013